### ISSUE TITLE
Ordering

### ISSUE THREAD

**Issue by arj03** [pietgeursen/patch-gatherings#6]
Hi

Not sure if this belongs in here or in patchbay, so feel free to direct me elsewhere.

Love this little feature in patchbay, but the ordering seems to be based on created in ascending order or something like date. Maybe a better ordering would be based on date of event. Maybe take distance of poster into account as well?

**Comment by pietgeursen** [comment]
Hi!,
Yeah thanks for this. It's on my list for sure.

Interesting you mention location. I've been thinking about this too. There isn't a way to know where a user is that posted the message. But I want to add the ability to publish about messages that give a gathering (or anything else) a location.

**Comment by arj03** [comment]
Yeah would be cool with a location on gatherings. The users location should always stay local and only be used for querying imo.
